core/http-auth-event-stream/pom.xml

```diff
@@ -44,6 +44,36 @@
             <artifactId>http-auth-spi</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>http-auth</artifactId>
```

Comment:
will this be replaced or will we add http-auth-aws when the V4 signers are moved to http-auth-aws

Reply:
We'll add it once they're moved.